blueocean-analytics-tools/src/main/resources/io/jenkins/blueocean/analyticstools/AnalyticsTools/header.jelly

```diff
@@ -0,0 +1,32 @@
+          ]]>
+        </script>
```

Comment:
🐛 This code def needs to go inside an IIFE, otherwise we're dropping things into the global scope, which is something we are desperately trying to stop Jenkins doing.